labelbox/data/serialization/labelbox_v1/label.py

```diff
@@ -146,6 +146,7 @@ class LBV1Label(BaseModel):
     skipped: Optional[bool] = Extra('Skipped')
     media_type: Optional[str] = Extra('media_type')
     data_split: Optional[str] = Extra('Data Split')
+    global_key: Optional[str] = Extra('Global Key')
```

Comment:
@kkim-labelbox added this into labels class. also recognize that many old tests did not include this in the past (part of the conversation about ever changing interfaces).